base/build.h

```diff
@@ -1,23 +0,0 @@
-// This file is a part of Julia. License is MIT: http://julialang.org/license
```

Comment:
this deletion looks unrelated, missed from a0e503ecec904e4ebc34e6529baa5912ed64ea61 ?